docs/rfc/rfc-1.adoc

```diff
@@ -184,22 +184,33 @@ reason), all they need to do is not submit their reveal and wait 30 days.
 
 [[sweeping]]
 ==== Sweeping
+
+Governable Parameters
+
+- `sweeping_refund_safety_time`: The amount of time prior to when a UTXO
+  becomes eligible where we will not include it in a sweeping transaction.
```

Comment:
prior to? 🤔 

My understanding is that:
- UTXO becomes eligible for sweeping after some time (protection against forks)
- UTXO is eligible for sweeping
- UTXO becomes not eligible for sweeping because `sweeping_refund_safety_time` is reached

Reply:
oops, forgot a word

Reply:
addressed in 0a50ade

Reply:
I think what @pdyraga was asking about here (Piotr, correct me if I'm wrong) was whether the `sweeping_refund_safety_time` is a value that's being added to the deposit creation time or being subtracted from the 30 days refund threshold...